core/src/main/java/com/orange/ouds/core/component/OudsDivider.kt

```diff
@@ -0,0 +1,141 @@
+ */
+@Composable
+fun OudsHorizontalDivider(
+    modifier: Modifier = Modifier,
+    color: OudsDivider.Color = OudsDivider.Color.Default
+) {
+    HorizontalDivider(modifier = modifier, color = color.value, thickness = OudsTheme.componentsTokens.divider.borderWidth.value)
+}
+
+//TODO Add DSM link when available
+// <a href="https://unified-design-system.orange.com/" class="external" target="_blank">OUDS Divider design guidelines</a>
+/**
+ * An OUDS vertical divider.
+ *
+ * This composable renders a vertical line that can be used to visually separate content.
+ * The color of the divider can be specified using the [OudsDivider.Color] enum, and the thickness is defined by the current theme's divider border width.
+ *
+ * @param modifier [Modifier] applied to the divider.
+ * @param color The color of the divider, chosen from the [OudsDivider.Color] enum. Defaults set to `OudsDivider.Color.Default`.
+ *
+ * @sample com.orange.ouds.core.component.samples.OudsVerticalDividerSample
+ */
+@Composable
+fun OudsVerticalDivider(
+    modifier: Modifier = Modifier,
+    color: OudsDivider.Color = OudsDivider.Color.Default
+) {
+    VerticalDivider(modifier = modifier, color = color.value, thickness = OudsTheme.componentsTokens.divider.borderWidth.value)
+}
+
+/**
+ * Contains classes to build an [com.orange.ouds.core.component.OudsHorizontalDivider] or an [com.orange.ouds.core.component.OudsVerticalDivider].
+ */
+object OudsDivider {
+
+    /**
+     * Represents the possible colors for an [OudsHorizontalDivider] or an [OudsVerticalDivider].
+     * Each color corresponds to a specific color key token from the design system.
+     */
+    enum class Color {
+        AlwaysBlack,
+        AlwaysOnBlack,
+        AlwaysOnWhite,
+        AlwaysWhite,
+        BrandPrimary,
+        Default,
+        Emphasized,
+        Muted,
+        OnBrandPrimary;
+
+        val value: androidx.compose.ui.graphics.Color
+            @Composable
+            get() {
+                return when (this) {
+                    AlwaysBlack -> OudsColorKeyToken.Always.Black
+                    AlwaysOnBlack -> OudsColorKeyToken.Always.OnBlack
+                    AlwaysOnWhite -> OudsColorKeyToken.Always.OnWhite
+                    AlwaysWhite -> OudsColorKeyToken.Always.White
+                    BrandPrimary -> OudsColorKeyToken.Border.BrandPrimary
+                    Default -> OudsColorKeyToken.Border.Default
+                    Emphasized -> OudsColorKeyToken.Border.Emphasized
+                    Muted -> OudsColorKeyToken.Border.Muted
+                    OnBrandPrimary -> OudsColorKeyToken.Border.OnBrand.Primary
+                }.value
+            }
+    }
+}
+
+internal enum class DividerOrientation {
+    Horizontal, Vertical
+}
+
+@UiModePreviews.Default
+@Composable
+private fun PreviewOudsHorizontalDivider(@PreviewParameter(OudsDividerPreviewParameterProvider::class) parameter: OudsDivider.Color) {
```

Comment:
`parameter` could be renamed to `color` (that's what we do when there is only one parameter that changes).